releasenotes/notes/profiling-stack-v2-task-names-1d566a1793280aa0.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Fixes an issue where ``asyncio`` task names are not properly propagated
+    when using stack v2, i.e. when ``DD_PROFILING_STACK_V2_ENABLED`` is set.
+    Fixes an issue where ``asyncio`` tasks are not associated with spans
```

Comment:
Copy/paste issue?

Reply:
Oh this PR fixes these two issues.

Reply:
Oh, sorry.  I saw the single `- |` and forgot how these yamls are parsed.  Carry on, apologies for the noise